### PR TITLE
Add question about applying Serenata to other instances

### PR DESCRIPTION
**Pull request by cuducos**
Add a topic to FAQ about using Serenata source code to other instances.

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -189,6 +189,16 @@ <h4 class="panel-title"><a aria-controls="collapseEighteen" aria-expanded="false
               </div>
             </div>
           </div>
+          <div class="panel panel-default">
+            <div class="panel-heading" id="headingNineteen" role="tab">
+                <h4 class="panel-title"><a aria-controls="collapseNineteen" aria-expanded="false" class="collapsed" data-parent="#accordion" data-toggle="collapse" href="#collapseNineteen" role="button">19. Posso usar o código da Serenata de Amor na minha cidade, no meu estado ou no meu páis?</a></h4>
```
Comment (Irio) [inline faq.html]: `páis` -> `país`

**Review by Irio — CHANGES_REQUESTED** [review]

File: faq.html (hunk 1 above)
Comment (cuducos) [reply faq.html]: Fixed by daa1425

**Review by Irio — APPROVED** [review]

**Comment by Irio** [comment]
After the translation to English, we also need to add the English version of this question/answer to the `_i18n` folder. Could you please add it?

**Comment by jtemporal** [comment]
>After the translation to English, we also need to add the English version of this question/answer to the _i18n folder. Could you please add it?

Isn't it already translated?

**Comment by cuducos** [comment]
> Isn't it already translated?

No, when I opened the PR there was no en-US version. One of these days I'll commit something new here ; )

**Comment by jtemporal** [comment]
hey @cuducos would love to merge this, can I hope for new commits and conflicts solving here? =)

**Comment by cuducos** [comment]
Closed in favor of #36

**Pull request closed without merging**